media-tv/plex-media-server/plex-media-server-1.16.4.ebuild

```diff
@@ -0,0 +1,157 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
(We are currently trying to get rid of python2-only software, you should REALLY start looking into getting python3 support for this, but it's not an issue now)

Reply:
Well, we can remove it from Gentoo, but upstream only supports 2.7, so I'm not sure what the approach should be. As the software isn't opensource, I don't think there's much we can do.

Reply:
I was afraid so. What I basically meant was, if you see patches for python3 support or if you find that other distros support python3 for this, implement that in Gentoo as well :P but yes, won't ask to do that now. Just something to keep an eye for.